src/promptflow-devkit/promptflow/_sdk/_service/apis/span.py

```diff
@@ -123,4 +123,4 @@ class Event(Resource):
     @api.response(code=200, description="Event")
     def get(self, event_id: str):
         client: PFClient = get_client_from_request()
-        return client._traces.get_event(event_id=event_id)
+        return client.traces.get_event(event_id=event_id)
```

Comment:
is there delete in pfs?

Reply:
not yet

Reply:
this change occurs just because i made `pf_client.traces` available in this pr...